portal-api/src/v1/gef/models/facilities.js

```diff
@@ -142,7 +143,7 @@ class Facility {
         this.guaranteeFee = req.guaranteeFee;
       }
 
-      if (req.ukefFacilityId !== null) {
+      if (req.ukefFacilityId && req.ukefFacilityId !== null) {
```

Comment:
you can simply run `if (req.ukefFacilityId) {...}`
this will check both for `undefined` and `null`
unless you need to check for `undefined`?

Reply:
Unfortunately simplifying the statement have a knock on effect, consulted with @ttbarnes and above condition is fail-safe.